src/pages/resultsView/survival/SurvivalHelper.tsx

```diff
@@ -0,0 +1,26 @@
+import jStat from 'jStat';
+
+export type ConfidenceIntervalSet = {
+    low: number;
+    high: number;
+};
+
+export function calculateLogConfidenceIntervals(
```

Comment:
what does it mean "log confidence interval"? why not "confidence interval"?

Reply:
There are several ways to calculate confidence interval, I found this may answer the question about `log confidence interval`: https://stats.stackexchange.com/questions/361354/choosing-conf-type-for-survfit-in-r/361387